LEAF_Request_Portal/sources/EmailTemplate.php

```diff
@@ -126,6 +126,24 @@ public function getEmailAndSubjectTemplateList()
         return $out;
     }
 
+    public function getCustomEmailTemplateList()
+    {
+        if (!$this->login->checkGroup(1)) {

```

Comment:
We need to start cutting down and fixing the bad coding of different return types.  May not be doable here, but we need to start shifting to only returning json with a "success" flag.